web/src/components/Review.js

```diff
@@ -36,9 +37,9 @@ const Review = ({
               // behave as intended on the outside.  Otherwise, the button
               // content can just be text.
               editHref ? (
-                <a href={editHref} ref={anchor}>
+                <Link to={editHref} ref={anchor}>
                   Edit
-                </a>
+                </Link>
```

Comment:
We should always use the Link element imported from 'react-router-dom' unless we are linking to someplace outside of the app.